kymatio/backend/base_backend.py

```diff
@@ -0,0 +1,62 @@
+
+
+class FFT:
+    def __init__(self, fft, ifft, irfft, sanity_checks):
+        self.fft = fft
+        self.ifft = ifft
+        self.irfft = irfft
+        self.sanity_checks = sanity_checks
+
+    def fft_forward(self, x, direction='C2C', inverse=False):
+        """Interface with torch FFT routines for 2D signals.
+
+            Example
+            -------
+            x = torch.randn(128, 32, 32, 2)
```

Comment:
This is very general, so add some more examples with the other backends

Reply:
ok